app/views/users/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% provide :title, t("sign_up") %>
+<h1 class="signup-title"><%= t("sign_up") %></h1>
+<div class="row">
+<div class="span6 offset3">
+  <%= form_for(@user) do |f| %>
+    <%= render 'shared/error_messages' %>
+  <% end %>
+</div>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
```

Comment:
So sánh params khi dùng form_for, form_tag, form_with em

Reply:
- Dùng form tag khi không có đối tượng, thường dùng cho form search, lọc
- Dùng form for cần có đối tượng để đặt vào vòng for, dùng cho tạo mới, sửa
- Dùng form with khi có đối tượng hoặc không có đối tượng đều được